### DISCUSSION TITLE
openaudible_home on Windows

### DISCUSSION THREAD

**Discussion by landreaus** [openaudible/openaudible#1162] [Q&A]
I was able to edit preferences in previous version without issue. I just upgraded to the new version today, and it overwrote my preferences. It's not pulling from the existing file that tells it that the data is to be stored on my e drive, and insists that I have no books because it is looking on my c drive. There is no openaudible_home file on my computer anywhere, including in the OpenAudible directory. It will not allow me to change it from the dialogue box within the program. I tried renaming the file that had my preferences from the old install to openaudible_home, and that didn't work.  How do I get this to work again?

**Comment by landreaus** [comment]
Some more coherent detail. I am editing the directories file. I copy in the directories from my previous install, which correctly point everything to the e drive and includes the OPENAUDIBLE_HOME SETTING pointing to e. I save and close the file. I open OpenAudible and it completely overwrites that file and sets it back to the default c drive settings. This is so irritating! I must be missing something, as the way that I'm doing it now worked with the previous install. I am going to uninstall the update until I can get an answer here.

**Comment by openaudible** [comment]
There is a new feature called Library Manager under the File menu. Use that to change or move your library.
The AUDIBLE_HOME setting is only if you don't want the preferences saved in your "C:\\Users\\USERNAME\\OpenAudible" directory.

**Comment by Cfomodz** [comment]
### Set Environment Variable

1. **Open System Properties**
   - Right-click "This PC" → Properties → Advanced system settings → Environment Variables

2. **Add User Variable**
   - Click "New..." under User variables
   - Variable name: `OPENAUDIBLE_HOME`
   - Variable value: `D:\MyOpenAudible` (your desired path)
   - Click OK

3. **Apply**
   - Close all OpenAudible instances completely
   - Restart OpenAudible
